my-app/src/components/Body/Content/BlogView/BlogView.jsx

```diff
@@ -0,0 +1,90 @@
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+import {Button} from "@material-ui/core";
+import useStyles from "./style";
+import {Typography} from "@material-ui/core";
+import Pagination from '@material-ui/lab/Pagination';
+import arrow from "../../../../assets/img/arrow-up-circle.svg";
+import PenToolWhite from "../../../../icons/PenToolWhite";
+
+const createData = (name, posted, status, stats, views, dots) => {
+    return {name, posted, status, stats, views, dots};
+};
+
+const rows = [
+    createData('Design: A Survival Guide for Beginners', 'Posted 3 days ago', "Scheduled", 120, "views", ""),
+    createData('Design: A Survival Guide for Beginners', 'Posted 3 days ago', 'Draft', 120, "views", ""),
+    createData('Design: A Survival Guide for Beginners', 'Posted 3 days ago', 'Scheduled', 120, "views", ""),
+    createData('Design: A Survival Guide for Beginners', 'Posted 3 days ago', 'Published', 0, "views", ""),
+    createData('Design: A Survival Guide for Beginners', 'Posted 3 days ago', 'Published', 120, "views", ""),
+];
+
+const BlogView = () => {
+    const classes = useStyles();
+
+    return (
+        <div className="blogView">
+            <TableContainer className={classes.tableContainer} component={Paper}>
+                <Table className={classes.table} aria-label="simple table">
+                    <TableHead>
+                        <TableRow>
+                            <TableCell className={classes.tableCell} align="left">Title</TableCell>
+                            <TableCell className={classes.tableCell} align="left">Status</TableCell>
+                            <TableCell className={classes.tableCell} align="left">Stats</TableCell>
+                            <TableCell className={classes.tableCell} align="right">
+                                <Button startIcon={<PenToolWhite/>} className={classes.button}>Add new</Button>
+                            </TableCell>
+                        </TableRow>
+                    </TableHead>
+                    <TableBody>
+                        {rows.map((row) => (
+                            <TableRow key={row.name} className={classes.row}>
+                                <TableCell component="th" scope="row" className={classes.cell}>
+                                    <Typography variant="h3" className={classes.name}>
+                                        {row.name}
+                                    </Typography>
+                                    <Typography variant="h5" className={classes.subtext}>
+                                        {row.posted}
+                                    </Typography>
+                                </TableCell>
+                                <TableCell align="center" className={classes.cell}>
+                                <span className={classes.status + " " + row.status}>
```

Comment:
what do you want here??
newer concat classes

Reply:
done